plugin/cassandra/plugin.go

```diff
@@ -554,16 +780,39 @@ func (p CassandraPlugin) Restore(endpoint plugin.ShieldEndpoint) error {
 		}
 		// Run sstableloader on each sub-directory found, assuming it is a table backup
 		tableDirPath := filepath.Join(keyspaceDirPath, tableDirInfo.Name())
-		cmd := fmt.Sprintf("sstableloader -u \"%s\" -pw \"%s\" -d \"%s\" \"%s\"", cassandra.User, cassandra.Password, cassandra.Host, tableDirPath)
+		cmd := fmt.Sprintf("%s/sstableloader -u \"%s\" -pw \"%s\" -d \"%s\" \"%s\"", DefaultBinDir, cassandra.User, cassandra.Password, cassandra.Host, tableDirPath)
 		plugin.DEBUG("Executing: `%s`", cmd)
 		err = plugin.Exec(cmd, plugin.STDIN)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "@R{\u2717 Load all tables data}\n")
 			return err
 		}
 	}
-	fmt.Fprintf(os.Stderr, "@G{\u2713 Load all tables data}\n")
+	return nil
+}
 
+func restoreUsers(cassandra *CassandraInfo, baseDir string) error {
+	plugin.DEBUG("Excluding cassandra user from 'system_auth.roles' table content")
+	cmd := fmt.Sprintf("sed -i -e '/^cassandra,/d' \"%s/system_auth.roles.csv\"", baseDir)
```

Comment:
This will fail if there is no `system_auth.roles.csv` file in the backup archive.  This causes issues restoring archives generated by current versions of this plugin, but it also proves problematic if an operator uses the new code, without saving users, and then decides to enable user-saving later.  All archives created before the switch will also break.

I believe we can just put a file check here (and elsewhere where new files are introduced) and behave accordingly.